acceptance/apps_test.go

```diff
@@ -215,6 +215,57 @@ configuration:
 		})
 	})
 
+	Describe("restage", func() {
+		When("restaging an app from git", func() {
+			It("will be staged again", func() {
+				wordpress := "https://github.com/epinio/example-wordpress"
+				pushLog, err := env.EpinioPush("",
+					appName,
+					"--name", appName,
+					"--git", wordpress+",main")
+				Expect(err).ToNot(HaveOccurred(), pushLog)
+
+				Eventually(func() string {
+					out, err := env.Epinio("", "app", "list")
+					Expect(err).ToNot(HaveOccurred(), out)
+					return out
+				}, "5m").Should(MatchRegexp(fmt.Sprintf(`%s.*\|.*1\/1.*\|.*`, appName)))
```

Comment:
We might have a helper function creating an app and waiting for it to be available.

Reply:
See acceptance/helpers/machine/apps.go

Reply:
Ah, much easier! Thanks